athenz-sia.env

```diff
@@ -61,9 +61,11 @@ INTERMEDIATE_CERT_BUNDLE=
 #
 # Mode to store/load backup of X.509 certificate to/from Kubernetes Secret
 # If write is specified, sia must be run uniquely per each secret to prevent conflict
-#	"read"
-#	"write"
-#	"read+write"
+# "": backup disabled
```

Comment:
I do not understand why indent jumps out when I have these on local as following:
![image](https://github.com/AthenZ/k8s-athenz-sia/assets/53258958/d34e364a-8ec3-4fc9-b7e7-fbf9c414b2a6)

Reply:
```suggestion
# "": 	 backup disabled
```

Reply:
Fixed!

Reply:
Hmm, I don't see it's fixed since there's still diff.